packages/calypso-e2e/src/lib/blocks/block-components/index.ts

```diff
@@ -0,0 +1 @@
+export * from './template-part-block';
```

Comment:
I created a new folder here where we can add blocks that follow this new pattern and aren't intended for smoke testing (aren't BlockFlows). We can migrate the old block POM classes here over time.

Reply:
To clarify, the idea is to move the current POMs under `calypso-e2e/src/lib/blocks` into here as they are updated?

Reply:
Oh, sure thing! How about `block-object-models`?

Reply:
>To clarify, the idea is to move the current POMs under calypso-e2e/src/lib/blocks into here as they are updated?

And yes, this is correct!